Project-3/NavdeepSingh/README.md

```diff
@@ -0,0 +1,58 @@
+# Weather Application
```

Comment:
Please add a link to your website and optionally screenshots of the functionality here itself.

Thanks!

Reply:
My code has an api key so is it ok to put the code publicly on github

Reply:
@ns-26 are you paying for the API key? If not you can host it publicly for now

Reply:
done.....do i need to create a new PR

Reply:
@ns-26 you can push a commit to the same branch, it will show up below